src/pages/Popup/Popup.tsx

```diff
@@ -28,12 +32,48 @@ const Popup = () => {
   }, [repo]);
 
   const sendActionToTabs = (action: any) => {
-    for (let i = 0; i < tabs.length; i++) {
-      const tabId = tabs[i].id as number;
+    tabs.filter(tabItem => (tabItem.isSelected)).forEach(tabItem => {
+      const tabId = tabItem.id as number;
       chrome.tabs.sendMessage(tabId, { action });
-    }
+    })
```

Comment:
I think you have to move this ')' up one row

Reply:
I've double-checked. This seems appropriate as it is.
```ts
tabs.filter(tabItem => (tabItem.isSelected)).forEach(tabItem => {
  const tabId = tabItem.id as number;
  chrome.tabs.sendMessage(tabId, { action });
})
```